jbpm-wb-process-runtime/jbpm-wb-process-runtime-client/src/main/java/org/jbpm/workbench/pr/client/editors/instance/list/ProcessInstanceListViewImpl.java

```diff
@@ -421,8 +440,56 @@ public void onClick( ClickEvent event ) {
 
         controlBulkOperations();
     }
+    
+    private Column<ProcessInstanceSummary, ProcessInstanceSummary> initErrorCountColumn() {
+        
+        Column<ProcessInstanceSummary, ProcessInstanceSummary> column = new Column<ProcessInstanceSummary, ProcessInstanceSummary>(
+                new AbstractCell<ProcessInstanceSummary>(){
+
+            @Override
+            public void render(Context context, ProcessInstanceSummary value, SafeHtmlBuilder sb) {
+                int errCount =  value != null ? value.getErrorCount() : 0;
+                ProcessRuntimeCSS css = ProcessRuntimeResources.INSTANCE.css();
+                String badgeHtml = null;
+                if(errCount > 0){
+                    //TODO: set link URL to correct error view perspective path
+                    String linkToErrorView = "#Error View?processInstanceId=" + value.getProcessInstanceId();
+                    String elementId = DOM.createUniqueId();
+                    String popoverAttrs = "data-toggle=\"popover\" data-html=\"true\" data-placement=\"right\"" +
```

Comment:
I realize I am late to the party (this PR already merged), but I'm not particularly fond of this kind of String-based markup generation. @rbarriuso @cristianonicolai Wouldn't it be better to reimplement it using Errai's DOM wrappers?